script/rhcos_versions.sh

```diff
@@ -1,34 +1,95 @@
 #!/usr/bin/env bash
 
-set -x
-set -e
```

Comment:
`set -o errexit -o pipefail` means exiting on any error (`errexit`) and treating errors in pipes as failures (`pipefail`). Solid settings.

Reply:
Done.